torchtune/utils/quantization.py

```diff
@@ -11,13 +11,15 @@
     apply_weight_only_int8_quant,
     Int4WeightOnlyGPTQQuantizer,
     Int4WeightOnlyQuantizer,
+    Int8DynActInt4WeightQuantizer,
     Quantizer,
 )
 
 __all__ = [
     "Int4WeightOnlyQuantizer",
     "Int4WeightOnlyGPTQQuantizer",
     "Int8WeightOnlyQuantizer",
+    "Int8DynActInt4WeightQuantizer",
```

Comment:
What are the benefits of this int8 dynamic act. quantizer over existing quantizers in torchtune? Do we plan to offer tutorials / additional documentation clarifying to users how to think about which quantizers to use?

Reply:
This is the only form of quantization supported by QAT right now, so this PR is adding the same form of quantization for PTQ for comparison. As for the documentation of which quantizer to use, I think we can add it in the future. Right now I'm not sure if we have any even in torchao or gpt-fast comparing 8da4w to weight only quantization. cc @jerryzh168 @HDCharles